vc_zoom/indico_vc_zoom/forms.py

```diff
@@ -11,7 +11,7 @@
 
 from wtforms.fields.core import BooleanField, StringField
 from wtforms.fields.simple import TextAreaField
-from wtforms.validators import DataRequired, ValidationError
+from wtforms.validators import DataRequired
```

Comment:
I don't think this change is correct - `_check_zoom_user` below uses this error!

Reply:
Whoops. Rebase problem.